lua/entities/terminator_nextbot_fakeply.lua

```diff
@@ -99,4 +99,25 @@ function ENT:GetAimVector()
     end
 
     return dir
-end
\ No newline at end of file
+end
+
+
+-- randomize bodygroups and skins
+function ENT:Initialize()
+    BaseClass.Initialize( self )
+    self:Appearance()
+end
```

Comment:
Should use the included "Additional" function here, also match style of the rest of the codebase
```suggestion
-- randomize bodygroups and skins
function ENT:AdditionalInitialize()
    self:Appearance()

end
```